### ISSUE TITLE
Find subject-matter experts to offer guidance

### ISSUE THREAD

**Issue by waldoj** [sunlightpolicy/State-Open-Data-Census#7]
We need somebody to agree to take each of these data categories, to provide feedback about which datasets are most important, what the hallmarks are of proper publication of data, and other assessment criteria. [These are the classes of data and the datasets that we have in mind](https://github.com/sunlightpolicy/State-Open-Data-Census/blob/master/Data/state%20open%20datasets%20-%20Sheet1.tsv). We need people to review and make recommendations on each of these:
- [ ] Companies
- [ ] Crime and Justice
- [ ] Earth Observation
- [ ] Education
- [x] Energy and Environment
- [ ] Finance and contracts
- [ ] Geospatial
- [ ] Economic Development
- [ ] Government Accountability and Democracy
- [ ] Health
- [ ] Science and Research
- [ ] Statistics
- [ ] Social Mobility and Welfare
- [ ] Transport and Infrastructure

We'll just comment every time somebody agrees to review a category (and if you are such a person, please feel free to comment!), and check off the box here so that we can see at a glance what's been claimed.

**Comment by fj** [comment]
Is there a deadline in mind?

I can volunteer for the "Energy and Environment" category. My _bona fides_ are a little out of date, but I helped build the [world's largest emissions inventory system](http://www.epa.gov/ttnchie1/eis/gateway/) back in 2007, which involved dealing with a lot of local, state, regional<sup>★</sup>, and federal data.

<sub>★ (EPA divides the country into ten arbitrary ["regions"](http://www2.epa.gov/aboutepa/visiting-regional-office) based on the location of their regional offices, so sometimes environmental data is aggregated at this level rather than at the state or federal level.)</sub>

**Comment by waldoj** [comment]
There's no deadline, but I think if folks didn't claim things within a week or so, we'd start muddling through on our own. :) Thanks for your help, @fj! @emily878 is putting together an instrument (#6). We'll try to keep it simple, basically just asking folks to volunteer their existing knowledge so we can get a good first-draft census in place.

**Comment by fj** [comment]
One followup: can you be a little more specific about what level of detail you need? I'm not certain from this issue how much effort is expected. For instance, are you expecting something that fits in a GitHub comment here in this thread, or are you looking for extensive, detailed commentary?

Put another way: How do I know if I've given you everything you need for my checkbox?

**Comment by emily878** [comment]
Hi John! I'm so happy you'll be helping out - I'm working on a
questionnaire for you to fill out, it should be up by Tuesday afternoon.

On Sun, Feb 15, 2015 at 3:23 AM, John Feminella notifications@github.com
wrote:

> One followup: can you be a little more specific about what level of detail
> you need? I'm not certain from this issue how much effort is expected. For
> instance, are you expecting something that fits in a GitHub comment here in
> this thread, or are you looking for extensive, detailed commentary?
> 
> Put another way: How do I know if I've given you everything you need
> (without having to ask you that explicitly)?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/sunlightpolicy/State-Open-Data-Census/issues/7#issuecomment-74408624
> .

## 

## 

Emily Shaw
National Policy Manager | Sunlight Foundation |
(o) 202-742-1520 x 282 | (c) 207-233-5684
@emilydshaw http://twitter.com/emilydshaw

**Comment by emily878** [comment]
@fj Thanks for getting involved! Here's the questionnaire to gather your feedback on the state-level datasets you've used: http://goo.gl/forms/aOa1Q8DWPd

**Comment by fj** [comment]
Hi @emily878 -- the survey seems to assume I've only ever used one dataset:

![survey screenshot](http://i.imgur.com/1vz6H4K.png)

since it's a list of radio buttons here and not a list of checkboxes. That's not really true, though; I've used at least six of those on that list (and a few that aren't on the list at all). Do you want me just to only focus on one dataset?

**Comment by emily878** [comment]
Excellent, thank you for the feedback @fj. Since we need information about individual datasets I tried to address this in the form intro: 

> Think of a single state-level dataset that you have used and then please fill out the following form. If 
> you're familiar with multiple state-level datasets, do go ahead and fill out this form as many times as 
> you'd like.

but I can put a shorter version of this on the main readme too. Can you think of a better way to gather information about multiple datasets? I'd be really happy to have something better, it was just the best I could come up with today. :smile:

**Comment by waldoj** [comment]
I have to confess that, while inelegant, your approach does seem like the simplest method. :)

**Comment by emily878** [comment]
@fj and @waldoj, I think the form was just too annoying! I changed it so that you can now check as many different datasets as you want within each issue area and I've put the data elements question on the same page so you don't have to flip back and forth. Check it out and please offer more suggestions if you have any!

**Comment by fj** [comment]
Can you check to see whether my previous submission a few days ago went
through? Just want to make sure I don't repeat myself.

**Comment by emily878** [comment]
Yes, they're in - and I made sure that the old form responses were saved
when I changed the form. Thanks again for your help so far!

On Mon, Feb 23, 2015 at 3:56 PM, John Feminella notifications@github.com
wrote:

> Can you check to see whether my previous submission a few days ago went
> through? Just want to make sure I don't repeat myself.
> 
> On Mon, Feb 23, 2015 at 12:05 PM, Emily Shaw notifications@github.com
> wrote:
> 
> > @fj https://github.com/fj and @waldoj https://github.com/waldoj, I
> > think the form was just too annoying! I changed it so that you can now
> > check as many different datasets as you want within each issue area and
> > I've put the data elements question on the same page so you don't have to
> > flip back and forth. Check it out and please offer more suggestions if
> > you
> > have any!
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/sunlightpolicy/State-Open-Data-Census/issues/7#issuecomment-75584575
> > 
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/sunlightpolicy/State-Open-Data-Census/issues/7#issuecomment-75629937
> .

## 

## 

Emily Shaw
National Policy Manager | Sunlight Foundation |
(o) 202-742-1520 x 282 | (c) 207-233-5684
@emilydshaw http://twitter.com/emilydshaw
